### PR TITLE
run page design

### PR DESCRIPTION
**Pull request by kikibubu05**

### CODE REVIEW

File: src/router/routes.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ const routes: RouteConfig[] = [
     component: () => import('layouts/MainLayout.vue'),
     children: [
       { path: '', component: () => import('pages/Move.vue') },
+      { path: 'Runpage', component: () => import('pages/Runpage.vue') }
```
Comment (Choreas) [inline src/router/routes.ts:9]: I'd like routes to be camelCased, it's more intuitive to me ^^

File: src/pages/Move.vue [hunk 2]
```diff
@@ -1,29 +1,42 @@
...
   name: 'Move',
-  components: { },
+  components: {},
 
   setup(props, {root}) {
-  
-    return {};
+
+    function runbtn(): void {
+      root.$router.push('Runpage')
```
Comment (Choreas) [inline src/pages/Move.vue:21]: It's fine to use 'runpage' as the name for this state of your screen. Please remember to change it to a more generic name once you extend the screen for the other two activities.

File: src/pages/Runpage.vue [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    </div>
+  </q-page>
+</template>
+
+<script lang="ts">
+import { defineComponent} from '@vue/composition-api';
+
+
+export default defineComponent({
+  name: 'Homepage',
```
Comment (Choreas) [inline src/pages/Runpage.vue:19]: The old name slipped through your checks ;)

File: src/pages/Runpage.vue [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    background-color:#E898A8;
+    position: absolute;
+    color: white;
+    top: 380px;
+    left: 203px;
+    height: 140px;
+    width: 168px;
+    border-radius: 30px;
+  }
+</style>
```
Comment (Choreas) [inline src/pages/Runpage.vue:78]: On larger screens, most of the screen will stay blank when you use absolute positions and scales in `px`.
Could you try to use relative values, e.g. percentage values? Inside css, you can simply set values in `%`, e.g. 
```
    top: 80%;
    left: 10%;
    height: 20%
    width: 80%;
```

File: src/pages/Move.vue (hunk 2 above)
Comment (kikibubu05) [reply src/pages/Move.vue:21]: like what?

Comment (Choreas) [reply src/pages/Move.vue:21]: Finally, run, cycle and walk will land on the same page, just differing in styling. So I'd say e.g. 'moveOngoing' or sth similar.

File: src/pages/Runpage.vue (hunk 4 above)
Comment (kikibubu05) [reply src/pages/Runpage.vue:78]: i have to change almost everything then

Comment (Choreas) [reply src/pages/Runpage.vue:78]: Sorry for the hassle, you could also leave it as it is for now. But I'd vote for changing it some time then.
We should generally avoid using absolute positioning throughout the app to render things nicely on all screens :D

File: src/pages/Runpage.vue [hunk 5]
```diff
@@ -16,7 +16,7 @@ import { defineComponent} from '@vue/composition-api';
 
 
 export default defineComponent({
-  name: 'Homepage',
+  name: 'Move',
```
Comment (Choreas) [inline src/pages/Runpage.vue:19]: You only forgot to change the name :)

**Comment by Choreas** [comment]
Looks great, I'm gonna merge it. Please remember to create the alternative design on figma when you got time and muse :)

**Pull request merged**